server/sonar-web/src/main/js/apps/issues/sidebar/CreationDateFacet.js

```diff
@@ -22,13 +22,13 @@ import React from 'react';
 import { max } from 'lodash';
 import { intlShape } from 'react-intl';
 import DateFromNow from '../../../components/intl/DateFromNow';
-import { longFormatterOption } from '../../../components/intl/DateFormatter';
+import { formatterOption, longFormatterOption } from '../../../components/intl/DateFormatter';
 import DateTimeFormatter from '../../../components/intl/DateTimeFormatter';
 import FacetBox from '../../../components/facet/FacetBox';
 import FacetHeader from '../../../components/facet/FacetHeader';
 import FacetItem from '../../../components/facet/FacetItem';
 import { BarChart } from '../../../components/charts/bar-chart';
-import DateInput from '../../../components/controls/DateInput';
+import DateRangeInput from '../../../components/controls/DateRangeInput';
 import { isSameDay, parseDate, toShortNotSoISOString } from '../../../helpers/dates';
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'toShortNotSoISOString'. ([javascript:S1128](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1128))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3157&issues=AWIqkLZunhKRZuqhdv-5&open=AWIqkLZunhKRZuqhdv-5)

[ISSUE_KEY]: AWIqkLZunhKRZuqhdv-5